### ISSUE TITLE
[mod-async] add support for CSS files from mediaPath

### ISSUE THREAD

**Issue by sirpreiss** [eightyfour/canny#4]
Hi, 
we're using the async module to load dynamic pages that contain further JS files to be loaded from a passed in mediaURL like 
```
canny.async.loadHTML(targetNode, {
                url :  config.mediaURL + 'index.html',
                mediaURL : config.mediaURL
            }, function () {
                console.log('Could not load');
            });
```

A sample index.html contains a relative path to the given resources from the mediaURL's point of view
```
<script type="text/javascript" src="./main.js"></script>
<div canny-mod="main"></div>
```

No we also want to load css files from that location via relative URL, but it fails, as the <link> elements are only attached to the targetNode without processing the src (href) URL.

**Comment by eightyfour** [comment]
Hey I'm working on it. What you could do is to parse your link tags by you own and replace the href attribute manually. E.g.:
```js
canny.async.loadHTML(targetNode, {
                url :  config.mediaURL + 'index.html',
                mediaURL : config.mediaURL
            }, onSuccess : function () {
                []slice.call(targetNode.getElementsByTagName('link'), function () {

});
            });

**Comment by eightyfour** [comment]
Hey,
I've fixed this issues with version 0.1.6.

Thanks for reporting the issue :)
